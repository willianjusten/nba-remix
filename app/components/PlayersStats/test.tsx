import { render } from '@testing-library/react'

import { PlayersStats } from '.'

describe('<PlayersStats>', () => {
  it('should render correctly', () => {
    const team = {
      tn: 'Team Name',
      tc: 'Team City',
      ta: 'Team Abbreviation',
      pstsg: [
        {
          fn: 'First',
          ln: 'Last',
          num: '2',
          min: '1',
          reb: '2',
          ast: '3',
          pts: '24',
        },
        {
          fn: 'Second',
          ln: 'Last',
          num: '1',
          min: '1',
          reb: '2',
          ast: '3',
          pts: '40',
        },
      ],
    }

    const { container } = render(<PlayersStats team={team} />)

    expect(container).toMatchSnapshot()
  })
})
