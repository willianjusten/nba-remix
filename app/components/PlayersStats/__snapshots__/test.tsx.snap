// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PlayersStats> should render correctly 1`] = `
<div>
  <div>
    <h1
      class="text-2xl font-bold"
    >
      Team City
       
      Team Name
    </h1>
    <table
      class="my-5 min-w-full border border-main bg-glass text-center text-white md:min-w-min"
    >
      <thead
        class="bg-slate-900 font-bold"
      >
        <tr>
          <td
            class="border border-main px-3 py-2 min-w-[120px] text-left sm:min-w-full"
          >
            Player
          </td>
          <td
            class="border border-main px-3 py-2 "
          >
            Min
          </td>
          <td
            class="border border-main px-3 py-2 "
          >
            Reb
          </td>
          <td
            class="border border-main px-3 py-2 "
          >
            Ast
          </td>
          <td
            class="border border-main px-3 py-2 "
          >
            Pts
          </td>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td
            class="border border-main px-3 py-2 truncate text-left"
          >
            F.
             
            Last
          </td>
          <td
            class="border border-main px-3 py-2 "
          >
            1
          </td>
          <td
            class="border border-main px-3 py-2 "
          >
            2
          </td>
          <td
            class="border border-main px-3 py-2 "
          >
            3
          </td>
          <td
            class="border border-main px-3 py-2 "
          >
            24
          </td>
        </tr>
        <tr>
          <td
            class="border border-main px-3 py-2 truncate text-left"
          >
            S.
             
            Last
          </td>
          <td
            class="border border-main px-3 py-2 "
          >
            1
          </td>
          <td
            class="border border-main px-3 py-2 "
          >
            2
          </td>
          <td
            class="border border-main px-3 py-2 "
          >
            3
          </td>
          <td
            class="border border-main px-3 py-2 "
          >
            40
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
`;
